core/src/test/scala/org/apache/spark/sql/delta/optimize/OptimizeMetricsSuite.scala

```diff
@@ -105,6 +107,43 @@ trait OptimizeMetricsSuiteBase extends QueryTest
     }
   }
 
+  test("optimize operation metrics in Delta table history") {
+    withSQLConf(DeltaSQLConf.DELTA_HISTORY_METRICS_ENABLED.key -> "true") {
+      withTempDir { tempDir =>
+        val sampleData =
+          0.to(79).seq ++ 40.to(79).seq ++ 60.to(79).seq ++ 70.to(79).seq ++ 75.to(79).seq
```

Comment:
I  dont fully understand whats happening here. what are u trying to achieve with this segmentation?
what is speciall about this 79 value?

Reply:
Nothing special about the data. I took the sample data generation from another test in the same suite.